environments/dev/backend_lb_config.yaml

```diff
@@ -12,6 +12,7 @@ gateways:
       - product: vh
         component: user-api
         ssl_enabled: true
+        health_path_override: /healthcheck/health
```

Comment:
it would be better to update the apps to use the standard ones, as otherwise you'll have to override this in every chart as well,

if this is just to temporarily unblock then fine 👍

Reply:
Just for testing, while Dev update code in all apps to use the standard ones 👍